Packs/MISP/Integrations/MISPV3/MISPV3.yml

```
@@ -2480,6 +2480,114 @@ script:
       description: Name of warninglists where the value was found.
       type: string
+  - arguments:
+    - description: The email address of the new user to be added.
+      name: email
+      required: true
+      type: textArea
+    - description: ID number indicating which organization the new user will be added to.
+      name: org_id
+      required: true
+      type: textArea
+    - description: Role of the new user to be added
+      name: role_id
+      required: true
+      type: textArea
+    - description: A password for the new user. Ensure that the password is at least 12 characters long, contains at least one upper-case, includes a digit or a special character, at least one lower-case character.
+      name: password
+      required: true
+    description: Adding a new user to MISP.
+    name: misp-add-user
+    outputs:
+    - contextPath: MISP.User.id
+      description: MISP User ID.
+      type: string
+    - contextPath: MISP.User.password
+      description: MISP User Password.
+      type: string
+    - contextPath: MISP.User.org_id
+      description: MISP User Organisation ID.
+      type: string
+    - contextPath: MISP.User.server_id
+      description: MISP User Server ID.
+      type: string
+    - contextPath: MISP.User.email
+      description: MISP User Email.
+      type: string
+    - contextPath: MISP.User.autoalert
+      description: MISP User Auto Alert.
+      type: boolean
+    - contextPath: MISP.User.authkey
+      description: MISP User Auth Key.
+      type: string
+    - contextPath: MISP.User.invited_by
+      description: MISP User Invited By.
+      type: string
+    - contextPath: MISP.User.gpgkey
+      description: MISP User GPG Key.
+      type: string
+    - contextPath: MISP.User.certif_public
+      description: MISP User Certif Public.
+      type: string
+    - contextPath: MISP.User.nids_sid
+      description: MISP User Nids Sid.
+      type: string
+    - contextPath: MISP.User.termsaccepted
+      description: MISP User Terms Accepted.
+      type: boolean
+    - contextPath: MISP.User.newsread
+      description: MISP User News Read.
+      type: string
+    - contextPath: MISP.User.role_id
+      description: MISP User Role ID.
+      type: string
+    - contextPath: MISP.User.change_pw
+      description: MISP User Change Password.
+      type: boolean
+    - contextPath: MISP.User.contactalert
+      description: MISP User Contact Alert.
+      type: boolean
+    - contextPath: MISP.User.disabled
+      description: MISP User Disabled.
+      type: boolean
+    - contextPath: MISP.User.expiration
+      description: MISP User Expiration.
+      type: string
+    - contextPath: MISP.User.current_login
+      description: MISP User Current Login.
+      type: string
+    - contextPath: MISP.User.last_login
+      description: MISP User Last Login.
+      type: string
+    - contextPath: MISP.User.force_logout
+      description: MISP User Force Logout.
```

```suggestion
      description: MISP user force logout.
```
Should this be "Whether the MISP user was forced to log out."